wiki/Help_Centre/Restricted_Mode/en.md

```diff
@@ -0,0 +1,58 @@
+
+If you have recently been granted an appeal and your account has become restricted again after a play, do not panic. This is something that happens to some appeal returnees. The support team will generally remove those restrictions within 24 hours. Feel free to send us an email or a tweet to catch our attention and be unrestricted faster!
+
+## Why am I restricted?
+
+There are not that many reasons why an account might be restricted.
+- **It is not your first account.** Multi-accounting is against the rules. Any accounts other than a person's first will become permanently restricted, but their original one will in most cases be left open and the user is encouraged to continue playing on it. If the multi-accounting is severe or abusive, a silence may be applied to their original account as a warning to cease creating new ones, and a restriction will be placed on the main account if the multi-accounting behaviour continues. If your account is restricted and you know you own other ones, please do not continue accessing the restricted accounts you have and stick to playing on your original one. If you have trouble accessing or verifying it, please send us an email at [accounts@ppy.sh](mailto:accounts@ppy.sh) so that we can help you recover access to it.
+- **You have allowed someone to play on your account.** Account sharing is also against the rules. If you've let someone use your account, it will be restricted, and the person who used it will likely receive a silence or be restricted themselves. If this happened accidentally, please let us know as soon as possible by sending an email to [accounts@ppy.sh](mailto:accounts@ppy.sh).
+- **You cheated.** Using a third-party program, macros, or any form of unfair advantage will result in an immediate restriction.
+- **Other cases.** The accounts team reserves the right to restrict an account for cases that are not covered by the three above. This generally includes investigating allegations of cheating or punishing excessive bad behaviour on the forums or chats.
+
+## What do I do while on cooldown before appealing?
+- **Keep enjoying the game!** While your account is restricted and you are on cooldown, you are more than welcome to continue playing the game (as long as you play on your original account!). The scores you achieve while restricted will be submitted, and unless your account is rolled back, they will remain after you become unrestricted.
+- **Maintain good behaviour.** We understand that being restricted can be frustrating, but think of it as a chance to take some time off, away from the game and community. Try to avoid interacting with the community at least for a few days (whether it be via discord or social media) to cool down and reflect on the actions that may have contributed to your account being restricted. Raging at others, spamming the support team, or insulting the staff is not going to help in making you unrestricted.
```

Comment:
`- **Maintain good behaviour.** We understand that being restricted can be frustrating, but think of it as a chance to take some time off, away from the game and community. Try to avoid interacting with the community for at least a few days (whether it be via Discord, social media, or other forms of contact) to cool down and reflect on the actions that may have contributed to your account being restricted. Raging at others, spamming the support team, or insulting the staff is not going to lift your restriction any faster.`